docset/winserver2022-ps/failoverclusters/Add-ClusterVirtualMachineRole.md

```diff
@@ -32,13 +32,15 @@ node, fails or requires scheduled maintenance, then another node begins to run t
 instead through a process known as failover. The virtual hard disk (VHD) file for the clustered
 virtual machine must be on the clustered disk used by that virtual machine.
 
-**Note:** This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP)
-authentication on the server computer.
-
-**Note:** This cmdlet generates a .TMP file under the user temp folder, and you can't have more than
-65535 .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist".
-For more information, please go to:
-https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344
+> [!NOTE]
+>
+> - This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP)
+> authentication on the server computer.
+>
+> - This cmdlet generates a .TMP file under the user temp folder, and you can't have more than 65535
+> .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For
```

Comment:
We should format file extensions as code.

```suggestion
> - This cmdlet generates a `.TMP` file under the user temp folder, and you can't have more than 65535
> `.TMP` files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For
```